### DISCUSSION TITLE
Captive portal disappears after first configuration SSID and after power down

### DISCUSSION THREAD

**Discussion by Bighoneypot** [Hieromon/AutoConnect#450] [Q&A] [answered]
Congratulations on this library.
I am using your library to send data on thingspeak, but I have a problem.
When I configure my device esp8266 chip using Captive Portal the captive portal disappears and I can no longer make changes.

In my serial monitor I have this :

/param.json open failed
WiFi connected:MywifiLan
IP:192.168.50.207
mDNS responder started

The device connects to the configured network but the captive portal disappears. Another problem is that it doesn't send data to the thingspeak channel.

I can no longer find the access point in the list of wireless networks.
Do you have any ideas?
I would like that the captive portal is always active, or that at each restart it turns on for at least a few minutes

Thank you.

**Comment by Hieromon** [comment, marked as answer]
That's normal behavior. You are mistaken about the behavior of the captive portal. If the ESP8266 device is transparent to the Internet (i.e. it has established a connection to one of the APs), your cell phone's OS will not automatically pop up.

> I can no longer find the access point in the list of wireless networks.

Use Configure New menu from [`/_ac`](https://hieromon.github.io/AutoConnect/menu.html#where-the-from). 

This topic is a Q&A, not a problem, so move it to the Discussions.
